lib/Components/Image/image.css

```diff
@@ -0,0 +1,48 @@
+
+.component-show-image-container{
+    display: flex;
+    justify-content: center;
```

Comment:
can make one class as i can see you are reusing it .

Reply:
yes refracting will be done afterwards!! thanks